managing-versions-and-packages.md

```diff
@@ -117,4 +114,13 @@ And to uninstall it:
 yarn remove react
 ```
 
+## What package manager should you use
+They both work well, so the choice between them is really up to you.
+
+If you want faster package installation and better error messages, try Yarn. If you want to use the same version of a package across different machines, use Yarn's lockfile feature.
+
+If you already know npm, stick with it. If you want a bigger community or you're more familiar with its commands, npm is a great choice too.
+
+[Here](https://www.geeksforgeeks.org/difference-between-npm-and-yarn/) is a cool article about the differences between them.
```

Comment:
I know it is written in the geeks resource, but I think it is worth to highlight these two points about yarn:
- "The intention behind developing yarn(at that time) was to fix performance and security concerns with npm."
- "Yarn generates a ‘yarn.lock’ file. Yarn lock files help in easy merge. The merges are predictable as well, because of the design of the lock file"